src/components/Card/CardReadMore.js

```diff
@@ -4,29 +4,40 @@ import classNames from 'classnames'
 
 import Icon from '../Icon/Icon'
 
+const tagPropType = PropTypes.oneOfType([
+  PropTypes.func,
+  PropTypes.string
+])
+
 const propTypes = {
   className: PropTypes.string,
   iconName: PropTypes.string,
   href: PropTypes.string,
-  text: PropTypes.string
+  text: PropTypes.string,
+  tag: tagPropType,
+  to: PropTypes.string
 }
 
 const defaultTypes = {
-  iconName: 'it-arrow-right'
+  iconName: 'it-arrow-right',
+  tag: 'a'
```

Comment:
The `tag:'a'` should be in defaultProps and not in defaultTypes. Please remove it here.